oap-server/server-starter/src/main/resources/ui-initialized-templates/rabbitmq/rabbitmq-cluster.json

```diff
@@ -732,7 +732,34 @@
       "entity": "Service",
       "name": "RabbitMQ-Cluster",
       "id": "RabbitMQ-Cluster",
-      "isRoot": false
+      "isRoot": false,
+      "isDefault": true,
+      "expressions": [
+        "avg(meter_rabbitmq_messages_published)*100",
+        "avg(meter_rabbitmq_messages_unconfirmed)*100",
+        "avg(meter_rabbitmq_queues)",
+        "avg(meter_rabbitmq_channels)",
+        "avg(meter_rabbitmq_connections)"
+      ],
+      "expressionsConfig": [
+        {
+          "unit": "%",
+          "label": "messages published rate"
+        },
+        {
+          "unit": "%",
+          "label": "messages unconfirmed rate"
```

Comment:
```
avg(meter_rabbitmq_messages_published)*100
avg(meter_rabbitmq_messages_unconfirmed)*100
```
What do those expressions mean？ I don't think they represent the percent rate.

The MAL are：
```
  - name: messages_published
    exp: rabbitmq_global_messages_received_total.sum(['cluster', 'node']).rate('PT1M')
  - name: messages_confirmed
    exp: rabbitmq_global_messages_confirmed_total.sum(['cluster', 'node']).rate('PT1M')
  - name: messages_routed
```

Reply:
Is that adding ```increase``` into the label ok?

Reply:
Following the original widget title is good. And remove `*100` from your expression.